setup.cfg

```diff
@@ -0,0 +1,116 @@
+# This file is used to configure your project.
+# Read more about the various options under:
+# http://setuptools.readthedocs.io/en/latest/setuptools.html#configuring-setup-using-setup-cfg-files
+
+[metadata]
+name = hmf
+description = A halo mass function calculator
+author = Steven Murray
+author-email = steven.g.murray@asu.edu
+license = mit
+long-description = file: README.rst
+long-description-content-type = text/x-rst; charset=UTF-8
+url = https://github.com/steven-murray/hmf
+project-urls =
+    Documentation = https://hmf.readthedocs.org
+# Change if running only on Windows, Mac or Linux (comma-separated)
+platforms = any
+# Add here all kinds of additional classifiers as defined under
+# https://pypi.python.org/pypi?%3Aaction=list_classifiers
+classifiers =
+    Development Status :: 6 - Mature
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Intended Audience :: Science/Research
+    License :: OSI Approved
+    License :: OSI Approved :: MIT License
+    Natural Language :: English
+    Topic :: Scientific/Engineering :: Physics
+    Topic :: Scientific/Engineering :: Astronomy
+
+[options]
+zip_safe = False
+packages = find:
+include_package_data = True
+package_dir =
+    =src
+# Add here dependencies of your project (semicolon/line-separated), e.g.
+install_requires =
+    numpy>=1.6.2
+    scipy>=0.12.0
+    astropy>=1.1
+    camb>=1.0.0<2.0
+
+[options.packages.find]
+where = src
+exclude =
+    tests
+
+[options.extras_require]
+# Add here additional requirements for extra features, to install with:
+# `pip install cal_coefficients[PDF]` like:
+# PDF = ReportLab; RXP
+# Add here test requirements (semicolon/line-separated)
+docs =
+    Sphinx>=1.7.5
+    numpydoc>=0.8.0
+    nbsphinx
+tests =
+    coverage>=4.5.1
+    pytest>=3.5.1
+    pytest-cov>=2.5.1
+    pre-commit
+    mpmath>=1.0.0
+    colossus>=1.2.1
+dev =
+    Sphinx>=1.7.5
+    numpydoc>=0.8.0
+    nbsphinx
+    coverage>=4.5.1
+    pytest>=3.5.1
+    pytest-cov>=2.5.1
+    pre-commit
+    mpmath>=1.0.0
+    colossus>=1.2.1
+
+;[options.entry_points]
```

Comment:
It is recommended to use console_scripts entrypoint instead of standalone in-repo scripts like you used to.